README.md

```diff
@@ -91,9 +91,27 @@ Development
 
 [![Build Status](https://travis-ci.org/hickford/MechanicalSoup.svg?branch=master)](https://travis-ci.org/hickford/MechanicalSoup)
 
-### Tests
+You can develop against multiple versions of Python using virtualenv:
+
+    python3 -m venv .virtual-py3 && source .virtual-py3/bin/activate
```

Comment:
Hmmm do you have a version of Python3 less than 3.3?
https://packaging.python.org/tutorials/installing-packages/#creating-virtual-environments

I think the only advantage of `python3 -m venv` is that you can skip the `pip3 install virtualenv` step.